apps/vr-tests-web-components/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
## [🕵 FluentUI-web-components Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28222&commitId=f4ccddad7d6b7b3127a7ec244d64f58bf30dcc5b&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 0 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.Accordion With Custom Icons RTL.normal.chromium.png|0| Added|
|Accordion.Accordion With Custom Icons.normal.chromium.png|0| Added|

</details>


<div id="vrtCommentFluentUI-web-components"/>

Reply:
These results of two added and two removed is expected because I ran the baseline pipeline before this PR and i've made changes to the PR examples since then